functions.sh

```diff
@@ -544,6 +544,34 @@ install_user_mode_emulation_binary() {
     install -m 755 -o root -g root ${EMULATOR} ${USR_BIN}
 }
 
+# Recursively copies the file structure from the passed directory to the chroot
+# environment.
+# Globals:
+#     CUSTOM_FILES
+# Arguments:
+#     Path to the transfered folder
+# Returns:
+#     None
+install_recur() {
```

Comment:
It's great you decided to put the function inside the `FS-related functions` section, but all of the functions in the section (and other sections) are arranged in alphabetical order. Please, put the function after `install_readonly`.